pkg/server/applications.go

```diff
@@ -0,0 +1,777 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/weaveworks/weave-gitops/pkg/server/middleware"
+
+	"github.com/benbjohnson/clock"
+	"github.com/weaveworks/weave-gitops/pkg/flux"
+	"github.com/weaveworks/weave-gitops/pkg/osys"
+	"github.com/weaveworks/weave-gitops/pkg/runner"
+	"github.com/weaveworks/weave-gitops/pkg/server/internal"
+	"github.com/weaveworks/weave-gitops/pkg/services"
+
```

Comment:
Can we please sort these imports? :))

Reply:
my linter doesn't complain :ahhhhh: what do you want to have changed? I always forget what the desired approach is :smile:

Reply:
:DDDD

stdlib
thridparty
organization
package-local

:D

Reply:
my linter doesn't agree with that completely, moving `flux` imports to below third party gets auto-moved

Reply:
https://github.com/weaveworks/weave-gitops/pull/1148/commits/0ae0b6c02cd02869f341bb009c0f5b9725fc6cb1